src/main/kotlin/gg/essential/elementa/components/inspector/CompactToggle.kt

```diff
@@ -0,0 +1,56 @@
+package gg.essential.elementa.components.inspector
+
+import gg.essential.elementa.components.UIBlock
+import gg.essential.elementa.components.Window
+import gg.essential.elementa.constraints.AspectConstraint
+import gg.essential.elementa.constraints.CenterConstraint
+import gg.essential.elementa.constraints.animation.Animations
+import gg.essential.elementa.dsl.*
+import gg.essential.elementa.state.State
+import gg.essential.elementa.utils.onLeftClick
+import gg.essential.elementa.utils.onSetValueAndNow
+import gg.essential.universal.USound
+import java.awt.Color
+
+internal class CompactToggle(
```

Comment:
As is, I've got absolutely no clue which way round is On vs Off. They look exactly the same, just mirrored.